### PR TITLE
Remove Bloodstone recipe to make it like vanilla

### PR DESCRIPTION
**Pull request by MasterGordon**

### CODE REVIEW

**Comment by Haganeko** [comment]
Wait, why is there a change to boss spawn.lua?

**Comment by MasterGordon** [comment]
Just had old stuff in here ups

**Comment by MasterGordon** [comment]
Travis pls

File: game/scripts/npc/items/item_bloodstone.txt [hunk 1]
```diff
@@ -8,14 +8,10 @@
   {
 
     "ID"                                                  "3400"       // unique ID
-    "Model"                                               "models/props_gameplay/recipe.vmdl"
-    "AbilityTextureName"                                  "item_recipe"
-    "BaseClass"                                           "item_datadriven"
```
Comment (chrisinajar) [inline game/scripts/npc/items/item_bloodstone.txt]: Why did you remove these lines? Add them back and everything will start working. You need a baseclass.
